scripts/model_zoo/update-models-from-caffe2.py

```diff
@@ -218,6 +218,7 @@ def onnx_verify(onnx_model, inputs, ref_outputs):
     'inception_v1': 'inception_v1',
     'inception_v2': 'inception_v2',
     'resnet50': 'resnet50',
+    'shufflenet': 'shufflenet',
```

Comment:
Why adding shufflenet back now?

Reply:
Because at the end we decide to keep the shufflenet in our model zoo, should update it.

Reply:
I see!